.eslintrc

```diff
@@ -0,0 +1,38 @@
+{
+  "env": {
+    "node": true
+  },
+  "globals": {},
+  "rules": {
+    "no-bitwise": 2,
+    "camelcase": 0,
+    "curly": 2,
+    "eqeqeq": 2,
+    "no-unused-expressions": [
+      2,
+      "allowTernary": true
```

Comment:
The extensionless version isn't a JSON object, but looking at the docs it looks like this format is deprecated now http://eslint.org/docs/user-guide/configuring#configuration-file-formats